From 5e2fb92f5373084caeb5bb1ade9bf04c1f25b901 Mon Sep 17 00:00:00 2001
From: Shakti Bhatnagar <shakti.bhatnagar@xilinx.com>
Date: Fri, 20 Mar 2015 19:06:10 +0530
Subject: [PATCH 1162/1851] mtd: Expand the ecc placement locations to 1260

commit 429fd27fc9da9b29f7812e18d93b398b26de6e27 from
https://github.com/Xilinx/linux-xlnx.git

Device like MT29F32G08ABCDBJ4 have a writesize/oobsize of 16K/1260 Bytes.
So, increasing the maximum ecc placement locations to 1260.

Value will be changed when mainline discussion is finished.

Signed-off-by: Shakti Bhatnagar <shaktib@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
Link: http://lists.infradead.org/pipermail/linux-mtd/2015-March/058398.html
State: not-upstreamable
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 include/linux/mtd/mtd.h | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/include/linux/mtd/mtd.h b/include/linux/mtd/mtd.h
index 936a3fdb48b5..0341b38c152d 100644
--- a/include/linux/mtd/mtd.h
+++ b/include/linux/mtd/mtd.h
@@ -72,7 +72,8 @@ struct mtd_oob_ops {
 };
 
 #define MTD_MAX_OOBFREE_ENTRIES_LARGE	32
-#define MTD_MAX_ECCPOS_ENTRIES_LARGE	640
+#define MTD_MAX_ECCPOS_ENTRIES_LARGE	1260
+
 /**
  * struct mtd_oob_region - oob region definition
  * @offset: region offset
-- 
2.31.1

